.bumpversion.cfg

```diff
@@ -8,6 +8,6 @@ search = Version = "{current_version}"
 replace = Version = "{new_version}"
 
 [bumpversion:file:README.md]
-search = go get -u github.com/watson-developer-cloud/go-sdk@v{current_version}
-replace = go get -u github.com/watson-developer-cloud/go-sdk@v{new_version}
+search = v{current_version}
```

Comment:
I changed this so that all instances of the current version within the README file will get changed, not just the one associated with the "Install" section.

Reply:
sexy